testfiles/emall.list

```diff
@@ -11,6 +11,7 @@ _BranchPumpsWithCommonPipe USA_IL_Chicago-OHare.Intl.AP.725300_TMY3
 _ConvCoefftest USA_FL_Tampa.Intl.AP.722110_TMY3
 _CoolingTowerDewPointRangeOp USA_IL_Chicago-OHare.Intl.AP.725300_TMY3
 _CoolingTowerWithDPDeltaTempOp USA_IL_Chicago-OHare.Intl.AP.725300_TMY3
+_CrashLights_9680 USA_IL_Chicago-OHare.Intl.AP.725300_TMY3
```

Comment:
I'm not sure that anyone uses `emall.list` anymore, but it's fine that it's added here.